### DISCUSSION TITLE
[WASM] MONO_WASM warning in Chrome console

### DISCUSSION THREAD

**Discussion by DierkDroth** [unoplatform/uno#13128] [Q&A]
I'm getting warning below in my Chrome console. This is pretty much latest UNO. 
```
dotnet.js:5 MONO_WASM: WebAssembly resource does not have the expected content type "application/wasm", so falling back to slower ArrayBuffer instantiation.
instantiate_wasm_asset @ dotnet.js:5
```

How could that warning be resolved? Thanks in advance

**Comment by DierkDroth** [comment]
I figured it out: I had to set the content-type to `application/wasm` when uploading any `*.wasm` file to my public AWS S3 bucket.

**Comment by scarny** [comment]
How do I resolve this warning? I have the same message in my webassembly application, the application is hosted on ngnix and I have already added the application/wasm wasm type in the /etc/nginx/mime.types file;

**Reply by jeromelaban** [reply]
Thanks for the question. It's likely that the configuration you have for nginx is not valid and/or not applied somehow. To validate that the mime type is right one, take a look at the browser's network tab for the `dotnet.wasm` request and make sure that it's what you expect it to be.

**Reply by scarny** [reply]
Thanks, in fact I see in the browser console that the Content-Type is not included in the .wasm files that are downloaded, which is instead included in all the other files.
For example, for a rest api request I have:

![image](https://github.com/unoplatform/uno/assets/2901048/ea7161c7-f644-45a7-ab71-0e63c0796589)

While for a .wasm file there is nothing like Content-Type:
![image](https://github.com/unoplatform/uno/assets/2901048/9dfd0850-08c0-48ee-a033-9ce36fcf201e)

But as mentioned I updated the ngnix mime-types to also include application/wasm wasm
so I don't know how to solve it
